### PR TITLE
Removing Log::info

### PR DESCRIPTION
**Pull request by joshbrw**
References PR #19

### CODE REVIEW

File: src/Stylist/Theme/Stylist.php [hunk 1]
```diff
@@ -125,9 +124,7 @@ public function activate($theme)
 
         $this->activeTheme = $theme;
 
-        $this->activateFinderPaths($theme);
-
-        Log::info("Using theme [{$theme->getName()}]");
+        $this->activateFinderPaths($theme;
```
Comment (simonfunk) [inline src/Stylist/Theme/Stylist.php]: hmm, does it not missing a clamp at the end? ;)

Comment (joshbrw) [inline src/Stylist/Theme/Stylist.php]: Not sure what happened there :s

Comment (kirkbushell) [inline src/Stylist/Theme/Stylist.php]: There's no issue - the tests would have broken if it were true. I've checked the file now and it's fine :)

**Comment by joshbrw** [comment]
There was a missing ), I fixed it and squashed the commits :)

**Comment by kirkbushell** [comment]
Oh. hehe

**Pull request merged**